apollo-router/tests/metrics_tests.rs

```diff
@@ -41,8 +41,8 @@ async fn test_metrics_reloading() -> Result<(), BoxError> {
         router.assert_reloaded().await;
     }
 
-    router.assert_metrics_contains(r#"apollo_router_cache_hit_count{kind="query planner",service_name="apollo-router",storage="memory"} 4"#, None).await;
-    router.assert_metrics_contains(r#"apollo_router_cache_miss_count{kind="query planner",service_name="apollo-router",storage="memory"} 2"#, None).await;
+    router.assert_metrics_contains(r#"apollo_router_cache_hit_count_total{kind="query planner",service_name="apollo-router",storage="memory",otel_scope_name="apollo/router",otel_scope_version=""} 4"#, None).await;
+    router.assert_metrics_contains(r#"apollo_router_cache_miss_count_total{kind="query planner",service_name="apollo-router",storage="memory",otel_scope_name="apollo/router",otel_scope_version=""} 2"#, None).await;
```

Comment:
There's also the lines below that need this change. When the env variables are set.

Reply:
Are you sure? I ran the tests and they all passed.

Reply:
done: https://github.com/apollographql/router/pull/3421/commits/8bdf325bcfad863ac604c3efea3fcc12a46545be